rust/operator/src/lib.rs

```diff
@@ -475,9 +471,14 @@ impl DruidState {
 
         let annotations = BTreeMap::new();
 
-        if let Some(plaintext_port) = plaintext_port {
-            cb.add_container_port(PLAINTEXT, plaintext_port.parse()?);
+        if let Some(plaintext_port) = validated_config
+            .get(&PropertyNameKind::File(RUNTIME_PROPS.to_string()))
+            .and_then(|props| props.get(DRUID_PLAINTEXTPORT))
+            .map(|port| port.parse::<i32>().unwrap())
+        {
+            cb.add_container_port(CONTAINER_PLAINTEXT_PORT, plaintext_port);
         }
+
```

Comment:
For our current monitoring approach the metrics port should be added here as well?

Reply:
Done.